coldfront/core/utils/management/commands/initial_setup.py

```diff
@@ -22,15 +22,17 @@ def handle(self, *args, **options):
             run_setup()
 
         else:
-            print(
-                """WARNING: Running this command initializes the ColdFront database and may modify/delete data in your existing ColdFront database. This command is typically only run once."""
+            self.stdout.write(
+                self.style.WARNING(
+                    """Running this command initializes the ColdFront database and may modify/delete data in your existing ColdFront database. This command is typically only run once."""
```

Comment:
Can we keep the `"WARNING:"`.. or add `"****WARNING****"`.. would like to keep some indication that this is a potential disastrous operation.